packages/eslint-plugin-pf-codemods/src/rules/helpers/JSXAttributes.ts

```diff
@@ -0,0 +1,18 @@
+import { JSXElement, JSXAttribute } from "estree-jsx";
+
+export function getAttribute(node: JSXElement, attributeName: string) {
+  const attributes = node.openingElement.attributes.filter(
+    (attr) => attr.type === "JSXAttribute"
+  ) as JSXAttribute[];
+  return attributes.find((attr) => attr.name.name === attributeName);
```

Comment:
Just a nit, is there a benefit in filtering the attributes first by type and then finding the one with correct name as opposed to just using single `find()`
```suggestion
  return node.openingElement.attributes.find((attr) => attr.type === "JSXAttribute" && attr.name.name === attributeName);
```

Reply:
Good catch! Thanks for the review

Reply:
oh, wait, something is failing